app/controllers/show-project.js

```diff
@@ -72,4 +77,52 @@ export default class ShowProjectController extends Controller {
       this.set('copySuccess', false);
     }, 2000);
   }
+
+  @action
+  handleFlagOpen() {
+    this.set('flagClosed', false);
+  }
+
+  @action
+  handleFlagClose() {
+    this.set('flagClosed', true);
+    // this.set('copySuccess', false);
+  }
+
+  @action
+  onCaptchaResolved(reCaptchaResponse) {
+    this.set('reCaptchaResponse', reCaptchaResponse);
+  }
+
+  @action
+  submitFlag() {
+    const projectname = this.get('model.dcp_projectname');
+    const projectid = this.get('model.dcp_name');
+    const flagText = this.get('flagText');
+    const reCaptchaResponse = this.get('reCaptchaResponse');
+
+    fetch(`http://localhost:3000/projects/feedback`, {
```

Comment:
This won't work in production... we should use environment variables for this.

Reply:
Done